src/EA.Weee.Web.Tests.Unit/Areas/Scheme/Controllers/HomeControllerTests.cs

```diff
@@ -1392,5 +1404,104 @@ public async void ChooseActivityGET_GivenOrganisationHasAesAndAatfsButAatfReturn
             model.PossibleValues.Should().NotContain(PcsAction.ManageAeReturns);
             model.PossibleValues.Should().NotContain(PcsAction.ManageAatfReturns);
         }
+
+        [Fact]
+        public async Task GetActivities_WithEnablePCSEvidenceNotesConfigurationSetToTrueAndOrganisationHasAnAatf_ReturnsPCSEvidenceNotesOption()
+        {
+            A.CallTo(() => weeeClient.SendAsync(A<string>._, A<GetOrganisationInfo>._))
+              .Returns(new OrganisationData
+              {
+                  SchemeId = A.Dummy<Guid>(),
+                  HasAatfs = true
+              });
+
+            var result = await HomeControllerSetupForPCSEvidenceNotes(true).GetActivities(A.Dummy<Guid>());
+
+            result.Should().Contain(PcsAction.ManagePcsEvidenceNotes);
+        }
+
+        [Fact]
+        public async Task GetActivities_WithEnablePCSEvidenceNotesConfigurationSetToTrueAndOrganisationHasNoAatf_DoesNotReturnPCSEvidenceNotesOption()
+        {
+            A.CallTo(() => weeeClient.SendAsync(A<string>._, A<GetOrganisationInfo>._))
+              .Returns(new OrganisationData
+              {
+                  HasAatfs = false
+              });
+
+            var result = await HomeControllerSetupForPCSEvidenceNotes(true).GetActivities(A.Dummy<Guid>());
+
+            result.Should().NotContain(PcsAction.ManagePcsEvidenceNotes);
+        }
+
+        [Fact]
+        public async Task GetActivities_WithEnablePCSEvidenceNotesConfigurationSetToFalseAndOrganisationHasAnAatf_DoesNotReturnsPCSEvidenceNotesOption()
+        {
+            A.CallTo(() => weeeClient.SendAsync(A<string>._, A<GetOrganisationInfo>._))
+              .Returns(new OrganisationData
+              {
+                  HasAatfs = true
+              });
+
+            var result = await HomeControllerSetupForPCSEvidenceNotes(false).GetActivities(A.Dummy<Guid>());
+
+            result.Should().NotContain(PcsAction.ManagePcsEvidenceNotes);
+        }
+
+        [Fact]
+        public async Task GetActivities_WithEnablePCSEvidenceNotesConfigurationSetToFalseAndOrganisationHasNoAatf_DoesNotReturnPCSEvidenceNotesOption()
+        {
+            A.CallTo(() => weeeClient.SendAsync(A<string>._, A<GetOrganisationInfo>._))
+              .Returns(new OrganisationData
+              {
+                  HasAatfs = false
```

Comment:
Give this a test a SchemeId, that way its checking when it has a scheme and the option is off then the option is not available